aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255_bulletproofs.move

```diff
@@ -129,6 +251,17 @@ module aptos_std::ristretto255_bulletproofs {
         num_bits: u64,
         dst: vector<u8>): bool;
 
+    /// Aborts with `error::invalid_argument(E_DESERIALIZE_RANGE_PROOF)` if `proof` is not a valid serialization of a
```

Comment:
It seems our move api accepts arbitrary bytes as `RangeProof`. In case of invalid serialization, better return `false` instead of aborting? just from API design perspective.

Reply:
Made `E_DESERIALIZE_RANGE_PROOF` deprecated

Reply:
Nope. Revert this. Backwards compatibility needs to be preserved.

Plus, abort code gives more info to caller about what went wrong.

Reply:
already reverted